script.js

```diff
@@ -0,0 +1,231 @@
+
+class Player {
+	constructor(name, symbol) {
+		this.name = name,
+		this.symbol = symbol
+	}
+}
+
+const Board = ( () => {
+	const boardArray = [null, null, null, null, null, null, null, null, null];
+	const winCombinations = [
+					[0,1,2],
+					[3,4,5],
+					[6,7,8],
+					[0,3,6],
+					[1,4,7],
+					[2,5,8],
+					[0,4,8],
+					[6,4,2]
+					];
+    
+	function render() {
+		const boardDiv = document.getElementById('board');
+		const cells = document.getElementsByClassName("sqr");
+		if (cells.length === 0){
+			for (let i=0; i<boardArray.length; i++) {
```

Comment:
When working with `JavaScript` arrays, it's generally better to use `forEach`, `for of`,`map` or other default methods than the usual `for` loop.

Reply:
Hi Ryanv,
I want to tell you that my partner and I did almost the suggestions that you made in your last review only the suggestion about the player wasn't clear  for us, you ask to store the player moves in the player object, in this case i will have an arrays for each player it means that i will have a lot of empty spots, can you be more explicit?
thanks.